packages/demo/public/index.html

```diff
@@ -5,6 +5,13 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
+    <script src="https://unpkg.com/react/umd/react.production.min.js" crossorigin></script>
```

Comment:
why is this here? shouldn't react be coming from package.json?

Reply:
i have no idea. I bet I was copy pasting the bootstrap stuff and accidentally grabbed this line too